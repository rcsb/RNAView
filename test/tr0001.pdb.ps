%!PS-Adobe-2.0
%%Title: A postscript generated from a RNAML file.
%%CreationDate: Fri Apr 16 11:33:11 2004
%%Orientation: Portrait
%%BoundingBox:     11   160   601   633

/Times-Bold findfont 12  scalefont setfont


/Times-Bold findfont 13  scalefont setfont    %%set font for (A U G C T)

%% stacking diagram widths for base-pairs 1 & 2
/W1 {1 setlinewidth} bind def
/W2 {1.5 setlinewidth} bind def
/W3 {2 setlinewidth} bind def
/W4 {3 setlinewidth} bind def

%% minor and major grooves filling color saturation
/MINOR_SAT 0.9 def
/MAJOR_SAT 0.1 def
/OTHER_SIDES {0 0 1 sethsbcolor fill} bind def

%% defineing geometry shapes
/NP {newpath} bind def
/CIRCLE {0 360 arc closepath} bind def                        %% circle 
/TRIANGLE {moveto lineto lineto closepath} bind def           %% triangle
/SQUARE {moveto lineto lineto lineto closepath} bind def      %% square
/LINE {moveto lineto stroke} bind def                         %% line
/DASHLINE { moveto lineto [2 4] 0 setdash stroke } bind def %% dashline
/DOTLINE  {1 0 1 setrgbcolor} bind def                        %% dotline
/Dw {1 setlinewidth} bind def
/FB {setgray fill} bind def
/R6 {moveto lineto lineto lineto lineto lineto closepath} bind def
/R9 {moveto lineto lineto lineto lineto lineto
     lineto lineto lineto closepath} bind def

%% line drawing colors for ACGITU & others
/Al {0.0000 1.00 1.00 sethsbcolor} bind def
/Cl {0.1667 1.00 1.00 sethsbcolor} bind def
/Gl {0.3333 1.00 1.00 sethsbcolor} bind def
/Il {0.3333 1.00 0.57 sethsbcolor} bind def
/Tl {0.6667 1.00 1.00 sethsbcolor} bind def
/Ul {0.5000 1.00 1.00 sethsbcolor} bind def
/Xl {0.0000 0.00 0.00 sethsbcolor} bind def
/XX {0.2000 0.20 0.20 sethsbcolor} bind def
/FILLBLACK {0.0000 0.00 0.00 sethsbcolor fill} bind def

%% minor groove filling colors for ACGITU & others
/Am {0.0000 MINOR_SAT 1.00      sethsbcolor fill} bind def
/Cm {0.1667 MINOR_SAT 1.00      sethsbcolor fill} bind def
/Gm {0.3333 MINOR_SAT 1.00      sethsbcolor fill} bind def
/Im {0.3333 MINOR_SAT 0.57      sethsbcolor fill} bind def
/Tm {0.6667 MINOR_SAT 1.00      sethsbcolor fill} bind def
/Um {0.5000 MINOR_SAT 1.00      sethsbcolor fill} bind def
/Xm {0.0000 0.00      MAJOR_SAT sethsbcolor fill} bind def

%% major groove filling colors for ACGITU & others
/AM {0.0000 MAJOR_SAT 1.00      sethsbcolor fill} bind def
/CM {0.1667 MAJOR_SAT 1.00      sethsbcolor fill} bind def
/GM {0.3333 MAJOR_SAT 1.00      sethsbcolor fill} bind def
/IM {0.3333 MAJOR_SAT 0.57      sethsbcolor fill} bind def
/TM {0.6667 MAJOR_SAT 1.00      sethsbcolor fill} bind def
/UM {0.5000 MAJOR_SAT 1.00      sethsbcolor fill} bind def
/XM {0.0000 0.00      MINOR_SAT sethsbcolor fill} bind def

%% define line width, line join style & cap style (1 means round)
1 setlinewidth 1 setlinejoin 1 setlinecap
    31   180 translate

/center{dup stringwidth pop 2 div neg 0 rmoveto } bind def
72.10 368.94  moveto (G) center show 
72.10 344.08  moveto (C) center show 
72.10 319.21  moveto (G) center show 
72.10 294.35  moveto (G) center show 
72.10 269.49  moveto (A) center show 
72.10 244.63  moveto (U) center show 
72.10 219.76  moveto (U) center show 
87.02 219.76  moveto (U) center show 
/center{dup stringwidth pop 2 div neg 0 rmoveto } bind def
gsave Al 101.94 219.76  moveto (A) center show grestore
291.11 136.97  moveto (g) center show 
266.25 136.68  moveto (C) center show 
241.38 136.40  moveto (U) center show 
216.52 136.12  moveto (C) center show 
195.77 150.11  moveto (A) center show 
170.77 151.34  moveto (G) center show 
/center{dup stringwidth pop 2 div neg 0 rmoveto } bind def
gsave Al 148.74 139.45  moveto (u) center show grestore
136.03 117.89  moveto (u) center show 
136.31 92.86  moveto (G) center show 
/center{dup stringwidth pop 2 div neg 0 rmoveto } bind def
gsave Al 149.50 71.59  moveto (G) center show grestore
171.80 60.21  moveto (G) center show 
196.76 62.00  moveto (A) center show 
217.20 76.45  moveto (G) center show 
242.06 76.74  moveto (A) center show 
266.92 77.02  moveto (G) center show 
291.78 77.30  moveto (C) center show 
341.50 77.86  moveto (g) center show 
366.37 78.14  moveto (C) center show 
391.23 78.42  moveto (C) center show 
416.09 78.71  moveto (A) center show 
440.95 78.99  moveto (G) center show 
465.81 79.27  moveto (A) center show 
490.67 79.55  moveto (c) center show 
516.65 72.55  moveto (U) center show 
540.58 84.86  moveto (g) center show 
550.00 110.06  moveto (A) center show 
540.01 135.04  moveto (A) center show 
515.81 146.80  moveto (g) center show 
490.00 139.22  moveto (A) center show 
465.13 138.94  moveto (P) center show 
440.27 138.65  moveto (c) center show 
415.41 138.37  moveto (U) center show 
390.55 138.09  moveto (G) center show 
365.69 137.81  moveto (G) center show 
340.83 137.53  moveto (A) center show 
131.77 170.04  moveto (G) center show 
/center{dup stringwidth pop 2 div neg 0 rmoveto } bind def
gsave Al 116.86 170.04  moveto (g) center show grestore
101.94 170.04  moveto (U) center show 
87.02 170.04  moveto (C) center show 
72.10 170.04  moveto (c) center show 
72.10 145.18  moveto (U) center show 
72.10 120.31  moveto (G) center show 
72.10 95.45  moveto (U) center show 
72.10 70.59  moveto (G) center show 
84.54 48.48  moveto (u) center show 
82.44 23.19  moveto (P) center show 
66.52 3.44  moveto (C) center show 
42.27 -4.00  moveto (G) center show 
/center{dup stringwidth pop 2 div neg 0 rmoveto } bind def
gsave Al 18.02 3.44  moveto (a) center show grestore
2.10 23.19  moveto (U) center show 
0.00 48.48  moveto (C) center show 
12.43 70.59  moveto (C) center show 
12.43 95.45  moveto (A) center show 
12.43 120.31  moveto (C) center show 
12.43 145.18  moveto (A) center show 
12.43 170.04  moveto (G) center show 
12.43 219.76  moveto (A) center show 
12.43 244.63  moveto (A) center show 
12.43 269.49  moveto (U) center show 
12.43 294.35  moveto (U) center show 
12.43 319.21  moveto (C) center show 
12.43 344.08  moveto (G) center show 
12.43 368.94  moveto (C) center show 
12.43 383.86  moveto (A) center show 
12.43 398.78  moveto (C) center show 
12.43 413.69  moveto (C) center show 
12.43 428.61  moveto (A) center show 
0.005 setlinewidth 1 setlinejoin 1 setlinecap
/Times findfont 11 scalefont setfont
/center{dup stringwidth pop 2 div neg 0 rmoveto } bind def
gsave Al 72.10 378.94 moveto (5') center show grestore
/Times findfont 11 scalefont setfont
/center{dup stringwidth pop 2 div neg 0 rmoveto } bind def
gsave Al 12.43 438.61 moveto (3') center show grestore
Il 72.10 372.94 72.10 348.08 LINE
 Il 72.10 348.08 72.10 323.21 LINE
 Il 72.10 323.21 72.10 298.35 LINE
 Il 72.10 298.35 72.10 273.49 LINE
 Il 72.10 273.49 72.10 248.63 LINE
 Il 72.10 248.63 72.10 223.76 LINE
 Il 72.10 223.76 87.02 223.76 LINE
 Il 87.02 223.76 101.94 223.76 LINE
 Il 101.94 223.76 291.11 140.97 LINE
 Il 291.11 140.97 266.25 140.68 LINE
 Il 266.25 140.68 241.38 140.40 LINE
 Il 241.38 140.40 216.52 140.12 LINE
 Il 216.52 140.12 195.77 154.11 LINE
 Il 195.77 154.11 170.77 155.34 LINE
 Il 170.77 155.34 148.74 143.45 LINE
 Il 148.74 143.45 136.03 121.89 LINE
 Il 136.03 121.89 136.31 96.86 LINE
 Il 136.31 96.86 149.50 75.59 LINE
 Il 149.50 75.59 171.80 64.21 LINE
 Il 171.80 64.21 196.76 66.00 LINE
 Il 196.76 66.00 217.20 80.45 LINE
 Il 217.20 80.45 242.06 80.74 LINE
 Il 242.06 80.74 266.92 81.02 LINE
 Il 266.92 81.02 291.78 81.30 LINE
 Il 291.78 81.30 341.50 81.86 LINE
 Il 341.50 81.86 366.37 82.14 LINE
 Il 366.37 82.14 391.23 82.42 LINE
 Il 391.23 82.42 416.09 82.71 LINE
 Il 416.09 82.71 440.95 82.99 LINE
 Il 440.95 82.99 465.81 83.27 LINE
 Il 465.81 83.27 490.67 83.55 LINE
 Il 490.67 83.55 516.65 76.55 LINE
 Il 516.65 76.55 540.58 88.86 LINE
 Il 540.58 88.86 550.00 114.06 LINE
 Il 550.00 114.06 540.01 139.04 LINE
 Il 540.01 139.04 515.81 150.80 LINE
 Il 515.81 150.80 490.00 143.22 LINE
 Il 490.00 143.22 465.13 142.94 LINE
 Il 465.13 142.94 440.27 142.65 LINE
 Il 440.27 142.65 415.41 142.37 LINE
 Il 415.41 142.37 390.55 142.09 LINE
 Il 390.55 142.09 365.69 141.81 LINE
 Il 365.69 141.81 340.83 141.53 LINE
 Il 340.83 141.53 131.77 174.04 LINE
 Il 131.77 174.04 116.86 174.04 LINE
 Il 116.86 174.04 101.94 174.04 LINE
 Il 101.94 174.04 87.02 174.04 LINE
 Il 87.02 174.04 72.10 174.04 LINE
 Il 72.10 174.04 72.10 149.18 LINE
 Il 72.10 149.18 72.10 124.31 LINE
 Il 72.10 124.31 72.10 99.45 LINE
 Il 72.10 99.45 72.10 74.59 LINE
 Il 72.10 74.59 84.54 52.48 LINE
 Il 84.54 52.48 82.44 27.19 LINE
 Il 82.44 27.19 66.52 7.44 LINE
 Il 66.52 7.44 42.27 0.00 LINE
 Il 42.27 0.00 18.02 7.44 LINE
 Il 18.02 7.44 2.10 27.19 LINE
 Il 2.10 27.19 0.00 52.48 LINE
 Il 0.00 52.48 12.43 74.59 LINE
 Il 12.43 74.59 12.43 99.45 LINE
 Il 12.43 99.45 12.43 124.31 LINE
 Il 12.43 124.31 12.43 149.18 LINE
 Il 12.43 149.18 12.43 174.04 LINE
 Il 12.43 174.04 12.43 223.76 LINE
 Il 12.43 223.76 12.43 248.63 LINE
 Il 12.43 248.63 12.43 273.49 LINE
 Il 12.43 273.49 12.43 298.35 LINE
 Il 12.43 298.35 12.43 323.21 LINE
 Il 12.43 323.21 12.43 348.08 LINE
 Il 12.43 348.08 12.43 372.94 LINE
 Il 12.43 372.94 12.43 387.86 LINE
 Il 12.43 387.86 12.43 402.78 LINE
 Il 12.43 402.78 12.43 417.69 LINE
 Il 12.43 417.69 12.43 432.61 LINE
 /Times findfont 8 scalefont setfont
/center{dup stringwidth pop 2 div neg 0 rmoveto } bind def
gsave Tl 1.43 319.21 moveto (70) center show grestore
gsave Tl 81.10 269.49 moveto (5) center show grestore
gsave Tl 1.43 170.04 moveto (65) center show grestore
gsave Tl 83.10 145.18 moveto (50) center show grestore
gsave Tl 291.01 145.96 moveto (10) center show grestore
gsave Tl 291.91 66.30 moveto (25) center show grestore
gsave Tl 441.07 67.99 moveto (30) center show grestore
gsave Tl 440.15 149.65 moveto (40) center show grestore
gsave Tl 168.11 163.04 moveto (15) center show grestore
gsave Tl 169.41 48.45 moveto (20) center show grestore
gsave Tl 562.00 110.19 moveto (35) center show grestore
gsave Tl 133.50 181.91 moveto (45) center show grestore
gsave Tl 93.58 18.74 moveto (55) center show grestore
gsave Tl -11.72 51.03 moveto (60) center show grestore
gsave Tl 24.43 413.69 moveto (75) center show grestore
NP W1
 Xl 64.10 370.94 20.43 370.94 LINE
 Xl 64.10 374.94 20.43 374.94 LINE
 NP W1
 Xl 64.10 346.08 20.43 346.08 LINE
 Xl 64.10 350.08 20.43 350.08 LINE
 NP W1
 Xl 64.10 321.21 20.43 321.21 LINE
 Xl 64.10 325.21 20.43 325.21 LINE
 NP W1
 Xl 42.27 298.35 3.00 CIRCLE 
 gsave  grestore stroke
 NP W1
 Xl 64.10 273.49 20.43 273.49 LINE
 NP W1
 Xl 64.10 248.63 20.43 248.63 LINE
 NP W1
 Xl 64.10 223.76 20.43 223.76 LINE
 NP W1
 Xl 93.76 219.45 134.66 193.25 LINE
 Xl 136.76 191.90 3.00 CIRCLE 
 gsave  grestore stroke
 Xl 138.87 190.55 143.92 187.32 LINE
 Xl 145.54 189.84 149.75 187.15 146.51 182.09 142.30 184.79 SQUARE
 gsave  grestore stroke
 Xl 148.13 184.62 189.03 158.42 LINE
 NP W1
 Xl 107.54 218.05 169.90 154.39 LINE
 Xl 172.04 156.49 176.24 152.21 171.96 148.01 167.76 152.29 SQUARE
 gsave  grestore stroke
 Xl 174.10 150.11 236.46 86.45 LINE
 NP W1
 Xl 293.20 132.99 293.69 89.32 LINE
 Xl 289.20 132.94 289.69 89.27 LINE
 NP W1
 Xl 268.34 132.71 268.83 89.04 LINE
 Xl 264.34 132.66 264.83 88.99 LINE
 NP W1
 Xl 241.48 132.40 241.97 88.73 LINE
 NP W1
 Xl 218.61 132.14 219.11 88.48 LINE
 Xl 214.61 132.10 215.11 88.43 LINE
 NP W1
 Xl 162.96 157.08 131.82 164.03 LINE
 Xl 128.89 164.69 3.00 CIRCLE 
 gsave  grestore stroke
 Xl 125.97 165.34 94.83 172.29 LINE
 NP W1
 Xl 142.47 138.48 81.69 90.29 LINE
 Xl 79.63 84.84 75.91 89.54 81.69 90.29 TRIANGLE
 gsave  grestore stroke
 Xl 77.77 87.19 73.07 83.46 LINE
 Xl 71.11 81.91 3.00 CIRCLE 
 gsave  grestore stroke
 Xl 69.15 80.35 8.37 32.16 LINE
 NP W1
 Xl 131.42 90.53 114.27 68.36 LINE
 Xl 112.74 66.38 3.00 CIRCLE 
 gsave  grestore stroke
 Xl 111.21 64.40 107.54 59.65 LINE
 Xl 109.91 57.82 105.17 61.49 104.48 55.70 TRIANGLE
 gsave  grestore stroke
 Xl 104.48 55.70 87.33 33.52 LINE
 NP W1
 Xl 144.59 68.97 73.97 10.97 LINE
 Xl 142.05 72.06 71.44 14.06 LINE
 NP W1
 Xl 211.35 85.91 172.88 121.79 LINE
 Xl 174.92 123.98 171.27 127.39 167.18 123.01 170.83 119.60 SQUARE
 gsave  grestore stroke
 Xl 169.22 125.20 164.83 129.29 LINE
 Xl 163.01 131.00 3.00 CIRCLE 
 gsave  grestore stroke
 Xl 161.18 132.70 122.71 168.58 LINE
 NP W1
 Xl 341.41 89.86 341.20 108.69 LINE
 Xl 341.17 111.69 3.00 CIRCLE 
 gsave FILLBLACK grestore stroke
 Xl 341.13 114.69 340.92 133.53 LINE
 NP W1
 Xl 368.28 90.16 367.78 133.83 LINE
 Xl 364.28 90.12 363.78 133.79 LINE
 NP W1
 Xl 393.14 90.45 392.64 134.11 LINE
 Xl 389.14 90.40 388.64 134.07 LINE
 NP W1
 Xl 416.00 90.70 415.50 134.37 LINE
 NP W1
 Xl 442.86 91.01 442.36 134.68 LINE
 Xl 438.86 90.96 438.36 134.63 LINE
 NP W1
 Xl 465.72 91.27 465.51 110.10 LINE
 Xl 465.47 113.10 3.00 CIRCLE 
 gsave FILLBLACK grestore stroke
 Xl 465.44 116.10 465.23 134.94 LINE
 NP W1
 Xl 64.10 172.04 20.43 172.04 LINE
 Xl 64.10 176.04 20.43 176.04 LINE
 NP W1
 Xl 64.10 149.18 20.43 149.18 LINE
 NP W1
 Xl 64.10 122.31 20.43 122.31 LINE
 Xl 64.10 126.31 20.43 126.31 LINE
 NP W1
 Xl 64.10 99.45 20.43 99.45 LINE
 NP W1
 Xl 64.10 72.59 20.43 72.59 LINE
 Xl 64.10 76.59 20.43 76.59 LINE
 NP W1
 Xl 77.91 47.99 57.90 34.44 LINE
 Xl 55.83 33.04 3.00 CIRCLE 
 gsave  grestore stroke
 Xl 53.76 31.64 48.79 28.27 LINE
 Xl 50.47 25.79 46.33 22.99 42.97 27.96 47.11 30.76 SQUARE
 gsave  grestore stroke
 Xl 44.65 25.47 24.64 11.92 LINE
 Al W2 285.23 142.18 137.65 172.82 gsave  DASHLINE grestore
 Al W2 490.60 89.55 490.06 137.22 gsave  DASHLINE grestore
 Al W2 193.34 70.93 90.45 218.84 gsave  DASHLINE grestore
 Al W2 260.89 143.39 107.29 221.06 gsave  DASHLINE grestore
 Al W2 537.91 133.42 518.75 82.17 gsave  DASHLINE grestore
 Al W2 9.49 69.36 2.94 57.71 gsave  DASHLINE grestore
 Al W2 136.25 102.86 136.10 115.89 gsave  DASHLINE grestore
 Al W2 546.01 109.57 520.64 81.03 gsave  DASHLINE grestore
 Al W2 15.79 13.01 2.23 46.90 gsave  DASHLINE grestore
 
 showpage 
