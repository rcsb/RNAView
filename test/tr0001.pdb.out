PDB data file name: tr0001.pdb
uncommon residue 2MG   10  on chain A [#10] assigned to: g
uncommon residue H2U   16  on chain A [#16] assigned to: u
uncommon residue H2U   17  on chain A [#17] assigned to: u
uncommon residue M2G   26  on chain A [#26] assigned to: g
uncommon residue OMC   32  on chain A [#32] assigned to: c
uncommon residue OMG   34  on chain A [#34] assigned to: g
uncommon residue  YG   37  on chain A [#37] assigned to: g
uncommon residue PSU   39  on chain A [#39] assigned to: P
uncommon residue 5MC   40  on chain A [#40] assigned to: c
uncommon residue 7MG   46  on chain A [#46] assigned to: g
uncommon residue 5MC   49  on chain A [#49] assigned to: c
uncommon residue 5MU   54  on chain A [#54] assigned to: u
uncommon residue PSU   55  on chain A [#55] assigned to: P
uncommon residue 1MA   58  on chain A [#58] assigned to: a
BEGIN_base-pair
     1_72, A:     1 G-C    72 A: +/+ cis         XIX
     2_71, A:     2 C-G    71 A: +/+ cis         XIX
     3_70, A:     3 G-C    70 A: +/+ cis         XIX
     4_69, A:     4 G-U    69 A: W/W cis         XXVIII
     5_68, A:     5 A-U    68 A: -/- cis         XX
     6_67, A:     6 U-A    67 A: -/- cis         XX
     7_66, A:     7 U-A    66 A: -/- cis         XX
     8_14, A:     8 U-A    14 A: W/H tran        XXIV
     9_23, A:     9 A-A    23 A: H/H tran   syn    II
    10_25, A:    10 g-C    25 A: +/+ cis         XIX
    11_24, A:    11 C-G    24 A: +/+ cis         XIX
    12_23, A:    12 U-A    23 A: -/- cis         XX
    13_22, A:    13 C-G    22 A: +/+ cis         XIX
    15_48, A:    15 G-C    48 A: W/W tran        XXII
    16_59, A:    16 u-U    59 A: S/W tran   syn    n/a
    18_55, A:    18 G-P    55 A: W/S tran        n/a
    19_56, A:    19 G-C    56 A: +/+ cis    syn    XIX
    22_46, A:    22 G-g    46 A: H/W tran     syn  VII
    26_44, A:    26 g-A    44 A: W/W cis         VIII
    27_43, A:    27 C-G    43 A: +/+ cis         XIX
    28_42, A:    28 C-G    42 A: +/+ cis         XIX
    29_41, A:    29 A-U    41 A: -/- cis         XX
    30_31, A:    30 G-A    31 A:      stacked
    30_40, A:    30 G-c    40 A: +/+ cis         XIX
    31_39, A:    31 A-P    39 A: W/W cis         n/a
    49_65, A:    49 c-G    65 A: +/+ cis         XIX
    50_64, A:    50 U-A    64 A: -/- cis         XX
    51_63, A:    51 G-C    63 A: +/+ cis         XIX
    52_62, A:    52 U-A    62 A: -/- cis         XX
    53_61, A:    53 G-C    61 A: +/+ cis         XIX
    54_58, A:    54 u-a    58 A: W/H tran     syn  XXIV
    69_70, A:    69 U-C    70 A:      stacked
    10_45, A:    10 g-G    45 A: H/S cis         !1H(b_b)
    32_38, A:    32 c-A    38 A: S/W cis         !1H(b_b)
     8_21, A:     8 U-A    21 A: S/W tran        !(b_s)
     9_11, A:     9 A-C    11 A: S/H cis    syn    !(b_s)
    33_36, A:    33 U-A    36 A: S/H tran        !b_(O1P,O2P)
    60_61, A:    60 C-C    61 A: S/H cis         !b_(O1P,O2P)
    17_18, A:    17 u-G    18 A: S/. tran        !(s_s)
    33_35, A:    33 U-A    35 A: S/S tran        !(s_s)
    58_60, A:    58 a-C    60 A: S/S tran   syn    !(s_s)
END_base-pair

Summary of triplets and higher multiplets
BEGIN_multiplets
9_12_23_| [1 3]  A: 9 A  +  A: 12 U  +  A: 23 A
13_22_46_| [2 3]  A: 13 C  +  A: 22 G  +  A: 46 g
END_multiplets
  The total base pairs =  30 (from   76 bases)
------------------------------------------------
 Standard  WW--cis  WW-tran  HH--cis  HH-tran  SS--cis  SS-tran
       20        3        1        0        1        0        0
  WH--cis  WH-tran  WS--cis  WS-tran  HS--cis  HS-tran
        0        3        0        2        0        0
------------------------------------------------
